app/src/main/res/layout/activity_home.xml

```diff
@@ -1,67 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    >
+    xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <data>
+    <data>
 
-    <variable
-        name="presenter"
-        type="com.udacity.nanodegree.blooddonation.ui.home.HomeActivityContract.Presenter"
-        />
-  </data>
+        <variable
```

Comment:
Again, HomeContract Presenter here.

Reply:
it is Home contract presenter ....do u want me to rename it?